src/Tvheadend.cpp

```diff
@@ -2231,17 +2254,28 @@ void CTvheadend::ParseRecordingAddOrUpdate ( htsmsg_t *msg, bool bAdd )
               continue;
 
             if (!htsmsg_get_u32(&stream->hmf_msg, "audio_type", &u32)) // Only present for audio streams
-              hasAudio = 1;
+              hasAudio = true;
             if (!htsmsg_get_u32(&stream->hmf_msg, "aspect_num", &u32)) // Only present for video streams
-              hasVideo = 1;
-            if (hasAudio && hasVideo) // No need to parse the rest
-              break;
+              hasVideo = true;
           }
         }
       }
-      rec.SetChannelType(hasVideo ? CHANNEL_TYPE_TV :
-          (hasAudio ? CHANNEL_TYPE_RADIO : CHANNEL_TYPE_OTHER));
+
+      if (!htsmsg_get_s64(&file->hmf_msg, "start", &s64) && (start == 0 || start > s64))
```

Comment:
Could be that I'm reading this the wrong way, but if a recording happens to be split into two files, shouldn't "start" get set to the value of the first part (i.e. `start < s64`, the opposite of what is done here)? Same with stop.

Reply:
I store  the last value of start, if I get a new start value, I check whee l whether the previous value is larger than the new value. If so, I store the be value. This, I should end up with start holding the lowest value when I'm done with iterating all files. Did I code this wrong?

Reply:
No you're right, I read it as `s64 > start` but obviously it's the other way around.